src/main/java/siri/Parser.java

```diff
@@ -0,0 +1,205 @@
+package siri;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalTime;
+
+class Parser {
+    TaskList taskList;
+
+    Parser(TaskList taskList) {
+        this.taskList = taskList;
+    }
+
+    //deals with making sense of the user command
+    
+    public int handleCommand(String s) throws SiriException{
```

Comment:
This is a long method. Is it better to shorten it using abstraction or other methods?